lib/node_modules/@stdlib/stats/base/dists/negative-binomial/stdev/benchmark/benchmark.js

```diff
@@ -32,16 +33,23 @@ var stdev = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var r;
 	var p;
 	var y;
 	var i;
 
+	len = 100;
+	r = new Float64Array( len );
+	p = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		r[ i ] = ceil( randu() * 100.0 );
+		p[ i ] = ( randu() * 1.0 ) + EPS;
```

Comment:
I realize this was present in the original as well, but can we please here and elsewhere remove the addition of `EPS` so `p` is guaranteed to fall into `[0,1]`?